### DISCUSSION TITLE
Test question from today's session

### DISCUSSION THREAD

**Discussion by shawnrhoads** [Center-for-Computational-Psychiatry/course_spice#1] [Q&A]
What is something surprising you discovered in today's session?

**Comment by fuqixiu** [comment]
**ALL** that *fancy* `markdown tricks`! and also Jupyter Book? (~~even not covered in class but~~ fire)

**Comment by chrismflores** [comment]
jupyter and github discussions

**Comment by leilaf1** [comment]
<font color = blue> As a novice programmer I have a lot of work to do!

**Comment by tvanteru** [comment]
learning basic functions while in cells!

**Comment by nykash** [comment]
Slicing a list even if the index is out of bounds might not give an error, just an empty list, so make sure to check that your indices are staying within bounds/acting as you want them to.
